packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -108,9 +109,15 @@ export class Linker extends Component<LinkerProps, LinkerState> {
     this.handleStateChange = this.handleStateChange.bind(this);
     this.handleExited = this.handleExited.bind(this);
     this.handleLinkSelected = this.handleLinkSelected.bind(this);
+    this.handlePanelDragging = this.handlePanelDragging.bind(this);
+    this.handlePanelDropped = this.handlePanelDropped.bind(this);
     this.isColumnSelectionValid = this.isColumnSelectionValid.bind(this);
 
-    this.state = { linkInProgress: undefined, selectedIds: new Set<string>() };
+    this.state = {
+      linkInProgress: undefined,
+      selectedIds: new Set<string>(),
+      showOverlay: true,
```

Comment:
I'd name this `isDraggingLinkedPanel` or even just `isDraggingPanel` instead, as it's kind of confusing naming it `showOverlay` when the overlay is shown based on the active tool in `isOverlayShown()` method